.gitignore

```diff
@@ -36,6 +36,9 @@ coverage.html
 # Sed backup files
 *.bak
 
+# JetBrains/GoLand/IntelliJ project file
+.idea
```

Comment:
This could go in your personal Git settings - I don't think we have any tool-specific ignores that aren't for tools specifically used in Weave Net.

Reply:
Okay, I'll remove it.